cloudify/dispatch.py

```diff
@@ -50,6 +50,12 @@
     serialize_known_exception,
     deserialize_known_exception
 )
+
+try:
+    from cloudify_agent.api.plugins import installer as plugin_installer
+except ImportError:
```

Comment:
keep this optional so that operations can be run without agent installed (tests/cfy local)